101-pubnub-eventhub-bridge/azuredeploy.json

```diff
@@ -0,0 +1,312 @@
+      }
+    },
+    "PubNub Egress Channel": {
+      "type": "String",
+      "defaultValue": "pnOutput",
+      "metadata": {
+        "description": "A channel name that the PubNub Publisher should publish back out on."
+      }
+    },
+    "PubNub Announce Channel": {
+      "type": "String",
+      "defaultValue": "pnAnnounce",
+      "metadata": {
+        "description": "A channel name that the PubNub Deployment script will alert on when the deployment has completed. If you don't intend on using this, just set this value to all caps, case-sensitive DISABLED. See below for more information on using the Provisioning Listener and the Announce channel."
+      }
+    },
+    "PubNub Publish Key": {
+      "type": "String",
+      "defaultValue": "demo-36",
+      "metadata": {
+        "description": "The PubNub Publish API Key that the PubNub component should publish against."
+      }
+    },
+    "PubNub Subscribe Key": {
+      "type": "String",
+      "defaultValue": "demo-36",
+      "metadata": {
+        "description": "The PubNub Subscribe API Key that the PubNub component should subscribe against."
+      }
+    },
+    "Azure Service Plan": {
+      "defaultValue": "USWestBasic",
+      "type": "String",
+      "metadata": {
+        "description": "Must be USWestBasic unless you ensure all components are in the same region.  See README for more information."
+      }
+    },
+    "Azure Ingress Event Hub Name": {
+      "defaultValue": "infromsubscriberhub",
+      "type": "String",
+      "metadata": {
+        "description": "The name you wish to give the Ingress (Input) Event Hub. You can accept the default, as the Event Hub name needs only to be unique within a unique Event Hub Namespace."
+      }
+    },
+    "Azure Egress Event Hub Name": {
+      "defaultValue": "outtopnpublisher",
+      "type": "String",
+      "metadata": {
+        "description": "The name you wish to give the Egress (Output) Event Hub. You can accept the default, as the Event Hub name needs only to be unique within a unique Event Hub Namespace."
+      }
+    },
+    "Azure Ingress SAS Policy Name": {
+      "defaultValue": "infromsubscriberhub",
+      "type": "String",
+      "metadata": {
+        "description": "The name you wish to give the Ingress (Input) Event Hub SAS Policy. You can accept the default, as the Event Hub SAS Policy name needs only to be unique within a unique Event Hub Namespace."
+      }
+    },
+    "Azure Egress SAS Policy Name": {
+      "defaultValue": "outtopublisherhub",
+      "type": "String",
+      "metadata": {
+        "description": "The name you wish to give the Egress (Output) Event Hub SAS Policy. You can accept the default, as the Event Hub SAS Policy name needs only to be unique within a unique Event Hub Namespace."
+      }
+    },
+    "apiVersion": {
+      "defaultValue": "2015-08-01",
```

Comment:
We generally recommend having hard-coded apiVersions. is there a reason to have it as a parameter? Do you expect the user to specify this frequently?